tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/SymbolPositionTest.java

```diff
@@ -147,6 +147,11 @@ public void testTypeDefPositions(int sLine, int sCol, String expSymbolName, int
             return new Object[][] {
                     {88, 4, "Module", 78, 12, 78, 18},
                     {80, 5, "TypeDef", 80, 5, 80, 12},
+                    {96, 4, "abc", 98, 17, 98, 20},
```

Comment:
Why add tests as position tests for these? 🤔

Reply:
When looking up a symbol of a documentation parameter, the actually defined position of the symbol is different from the cursor position. 
Therefore, in order to verify that info as well, decided to use the Position Tests for these.

The first two parameters are the cursor position, followed by the symbol name, and the position where the symbol is actually defined.

Reply:
I see. I think it'd be best to move these tests to their own test class since what we are primarily interested in is whether `symbol()` works for doc params as well. This position test is for testing particular scenarios where we need to check whether the symbol's position is correct.

Reply:
@pubudu91 The https://github.com/ballerina-platform/ballerina-lang/blob/master/tests/ballerina-compiler-api-test/src/test/resources/test-src/documentation_test.bal source file contains a couple of compilation errors that block the `CompilerPhaseRunner` from proceeding after the Type Checking phase. 

Therefore, would it be better if a separate source file is created if the `DocumentationTest` class is to be used for these tests as well?

Reply:
Moved to `DocumentationTest` class.